Gemfile.lock

```diff
@@ -584,15 +584,15 @@ GEM
       json
     redcarpet (3.5.1)
     redis (4.2.5)
-    redis-actionpack (5.1.0)
-      actionpack (>= 4.0, < 7)
-      redis-rack (>= 1, < 3)
+    redis-actionpack (5.2.0)
```

Comment:
we use this through `redis-rails`, adds support for signed encrypted cookies, see https://github.com/redis-store/redis-actionpack/blob/master/CHANGELOG.md#v520-2020-01-13